app/src/main/java/com/teamagam/gimelgimel/app/map/esri/EsriGGMapView.java

```diff
@@ -417,6 +431,18 @@ private LocationDisplayer getLocationDisplayer(Context context) {
     return new LocationDisplayer(getLocationDisplayManager(), locationListener);
   }
 
+  private OnTouchListener getPannableMapTouchListener() {
+    return new MapDragEventsEmitterTouchListenerDecorator(
```

Comment:
I want the observable to be working regardless of the panning mode.
Imagine someone observing it and not getting events because the panning mode is enabled >.<